build.gradle

```diff
@@ -9,6 +9,7 @@ plugins {
 dependencies {
 	implementation 'com.acrolinx.client:sdk:1.0.22'
 	implementation 'org.slf4j:slf4j-api:2.0.9'
+	implementation 'com.google.guava:guava:32.1.3-jre'
```

Comment:
Our goal is to reduce the number of external dependencies, therefore I would like to not add Guava here just to read the resource file into a byte array.

Since we have to be Java8 compatible, we can for example do it like this:
> byte[] content = Files.readAllBytes(Paths.get(SdkDemo.class.getResource(wordDocumentName).toURI()));

Btw, once we upgrade to Java 11, we can then further simplify this, e.g.:
> byte[] content = SdkDemo.class.getResourceAsStream(wordDocumentName).readAllBytes();